src/components/Profile/ProfileImageField.jsx

```diff
@@ -66,42 +56,45 @@ const ProfileImageField = ({ loadProfileData }) => {
       sx={{
         display: 'flex',
         flexDirection: 'column',
-        maxWidth: '300px',
-        alignItems: 'flex-start',
+        justifyContent: 'center',
+        alignItems: 'center',
+        boxShadow: '0 0 3px 0 black',
+        padding: '20px',
         gap: '10px'
       }}
     >
       <Typography color="black">Profile Image: </Typography>
       <Avatar
-        src={profileImg}
+        src={clientProfile ? clientProfile?.profileImg : profileImg}
```

Comment:
if `clientProfile` is `true` do we need `clientProfile?.profileImg` - could it just be `clientProfile.profileImg` ?